src/main/java/com/google/cloud/tools/maven/cloudsdk/CloudSdkMojo.java

```diff
@@ -64,8 +64,8 @@ public String getCloudSdkVersion() {
     return cloudSdkVersion;
   }
 
-  public File getServiceAccountKeyFile() {
-    return serviceAccountKeyFile;
+  public Path getServiceAccountKeyFile() {
```

Comment:
Could it be that there was a reason to use `File` instead of `Path`? I vaguely remember Maven did not support `File` somewhere, but that may be irrelevant here.